### PR TITLE
gist, vboxmanage-cloud: fix typos

### PR DESCRIPTION
**Pull request by xiaoxiangxianzi**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, welcome to tldr and thanks for the fixes.

File: pages/common/gist.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > Upload code to https://gist.github.com.
 > More information: <https://github.com/defunkt/gist>.
 
-- Log in in gist on this computer:
+- Log in gist on this computer:
```
Comment (vitorhcl) [inline pages/common/gist.md:6]:
```suggestion
- Log in to gist on this computer:
```
The second "in" was being used in the sense of "to", so I think it's better to use "to"

**Review by vitorhcl — APPROVED** [review]
Thank you! LGTM after we discuss my suggestion

**Comment by vitorhcl** [comment]
Just a minor tip: when we modify just a few pages, like 2 to 3, we generally put their names before the semicollon in the commit messages and in the PR title. See the recommended template:
> For the commit message and PR title of page changes, use the following format:
> 
> `{{command}}: type of change`
> Where `{{command}}` is the name of the command being modified, and `type of change` can be (but not limited to) one of the following examples:
> ...
> - For related changes to some pages: `grep, find, locate: synchronize format of wildcards`
> - For related changes to several unrelated pages: `pages*: fix Linux casing`
>
> ...

File: pages/common/gist.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 > Upload code to https://gist.github.com.
```
Comment (acuteenvy) [inline pages/common/gist.md:3]:
```suggestion
> Upload code to <https://gist.github.com>.
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**